guides/06_Meeting_Events.md

```diff
@@ -248,6 +252,22 @@ JavaScript uses the browser's [`getUserMedia` API](https://developer.mozilla.org
 
 <br>
 
+### Audio transmission related events
+
+The Javascript SDK publishes the following events to report issues with audio transmission:
```

Comment:
I believe it should always be `Chime SDK for JavaScript` .

Reply:
'audio transmission and receipt'?

Reply:
Just to add to Henry's first comment: The Amazon Chime SDK for JavaScript :)